### PR TITLE
Update RAC Calendar docs: change visibleDuration months from 2 to 3

### PR DESCRIPTION
**Pull request by reidbarber**
From testing session.

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

### CODE REVIEW

**Review by snowystinger — APPROVED** [review]

File: packages/react-aria-components/docs/Calendar.mdx [hunk 1]
```diff
@@ -422,7 +422,7 @@ The error message element within a `Calendar` can be targeted with the `[slot=er
 Multiple `CalendarGrid` elements can be rendered to show multiple months at once. The `visibleDuration` prop should be provided to the `Calendar` component to specify how many months are visible, and each `CalendarGrid` accepts an `offset` prop to specify its starting date relative to the first visible date.
 
 ```tsx example
-<Calendar aria-label="Appointment date" visibleDuration={{months: 2}}>
+<Calendar aria-label="Appointment date" visibleDuration={{months: 3}}>
```
Comment (snowystinger) [inline packages/react-aria-components/docs/Calendar.mdx:425]: does this actually show 3 months? I'm only seeing two, I think we need to include a calendargrid down below with a month offset -1?

**Review by snowystinger — CHANGES_REQUESTED** [review]

File: packages/react-aria-components/docs/Calendar.mdx (hunk 1 above)
Comment (reidbarber) [reply packages/react-aria-components/docs/Calendar.mdx:425]: Ahh good catch!

Comment (snowystinger) [reply packages/react-aria-components/docs/Calendar.mdx:425]: Our other calendars have the center month as the current one, and then -1 and +1 on either side? or am I interpreting that differently?

**Review by devongovett — APPROVED** [review]

File: packages/react-aria-components/docs/Calendar.mdx (hunk 1 above)
Comment (reidbarber) [reply packages/react-aria-components/docs/Calendar.mdx:425]: I don't think we do that anywhere. Not sure what the difference would be, but I'm happy to update that if its more accurate.

**Review by LFDanLu — APPROVED** [review]

File: packages/react-aria-components/docs/Calendar.mdx (hunk 1 above)
Comment (snowystinger) [reply packages/react-aria-components/docs/Calendar.mdx:425]: ah, yep, i was mistaken, i thought they were centered, they are not

**Review by snowystinger — APPROVED** [review]

**Pull request merged**